src/hooks/subscription-hooks.js

```diff
@@ -76,6 +77,33 @@ export function useJurorBalancesSubscription(jurorId) {
       movements = [],
     } = jurorData.juror || {}
 
+    const movementsFiltered = movements => {
+      // If the juror is activating from the wallent we need to just show an incoming movement in the Active balance
+      return movements.reduce((previousList, movement, index, originalList) => {
```

Comment:
Should we rename `previousList` to something with more meaning? 
Was thinking something like `accumulatorList`

Reply:
I think we could use only one condition (also taking your suggestions @fabriziovigevani):

```jsx
  return movements.reduce((movements, movement, index, prevMovements) => {
    if (
      index > 0 &&
      ANJMovement[prevMovements[index - 1].type] === ANJMovement.Activation &&
      ANJMovement[movement.type] === ANJMovement.Stake &&
      prevMovements[index - 1].createdAt === movement.createdAt
    ) {
      movements[movements.length - 1].type = 'StakeActivation'
      return movements
    }

    movements.push({
      ...movement,
      effectiveTermId: parseInt(movement.effectiveTermId, 10),
      amount: bigNum(movement.amount),
    })

    return movements
  })

```